compliance/cis_k8s/rules/cis_1_1_1/data.yaml

```diff
@@ -0,0 +1,44 @@
+metadata:
+  uuid: 41308bcdaaf665761478bb6f0d745a5c
```

Comment:
hey @jeniawhite, please update the `csp_rule` in the [technical index](https://github.com/elastic/security-team/blob/main/docs/cloud-security-posture-team/Technical_Index.md#rule-schema-finding) to use `uid` instead of `id`

Reply:
on the contrary, please leave it `id`. that would also match rule ecs that we plan to support in the future.